inspirehep/modules/workflows/views/callback.py

```diff
@@ -97,7 +148,13 @@ def webcoll_callback():
 
     Expects the request data to contain a list of record ids in the
     recids field.
+
+    DEPRECATED: replaced by the simpler continue_workflow_callback
```

Comment:
btw. that change requires changing the inspire-crawler code to use the robotupload callback instead of continue_workflow right? Or that is part of another flow?